### DISCUSSION TITLE
why does lazer not have audio offset key binds?

### DISCUSSION THREAD

**Discussion by PMorghK** [ppy/osu#22439] [Q&A] [answered]
So I was having a discussion with members of the community and they brought up some of the stuff that stable does better than lazer currently. which made me want to make a discussion about the features/problems discussed so that hopefully they will be resolved at least in some way.

The topic that was brought up was: why does lazer not allow audio offset to be changed with the + and - keys. This feature is in stable and some players heavily rely on it. They also rely on the feature existing during breaks in the map. So if they are pressing early or late maybe in the last section, they could adjust it so that they will play on time in the next section. 

This technically is a non issue since there should be no need to adjust offset mid way through the map but this might be untrue when considering longer maps (in the 10 to 20 minute range). Also generally this question is specifically about why the keys + and - are not bound at all. 

thank you for your time and hope you have a good day/night.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/21882

**Reply by PMorghK** [reply]
I should really not make discussions after a heated argument. sorry bdach for the duplicate.
